.github/workflows/build.yml

```diff
@@ -174,12 +135,15 @@ jobs:
             -o yaml \
             < ${GITHUB_WORKSPACE}/.signadot/frontend-template.yaml \
             > ${GITHUB_WORKSPACE}/.signadot/frontend-response.yaml
+
       - name: Test Sandbox
         run: |
           go install github.com/mikefarah/yq/v4@latest
           endpoint=$(yq '.endpoints[0].url' ${GITHUB_WORKSPACE}/.signadot/frontend-response.yaml)
           curl -H "signadot-api-key: ${{ secrets.SIGNADOT_API_KEY }}" $endpoint
+
       - name: Delete Sandbox
+        if: steps.create-sandbox.outcome == 'success'
```

Comment:
This new line is important because we were leaving orphan sandboxes in the demo cluster whenever the test failed.